packages/common/integration/tests/string/filter.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { filter } from '@angular-ru/common/string';
+
+describe('[TEST]: Filter', () => {
```

Comment:
do you have tests for `char === ' ' || char === '\\s'` cases

Reply:
added tests
`
expect(filter(' a b  c  ', ['a', 'b', 'c'])).toEqual('abc');
expect(filter(' a b  c  ', ['a', 'b', 'c', '\\s'])).toEqual('abc');
expect(filter(' a b  c  ', ['a', 'b', 'c', ' '])).toEqual(' a b  c  ');
`